@xen-orchestra/lite/src/stores/alarm.store.ts

```diff
@@ -0,0 +1,23 @@
+import { useXapiCollectionStore } from "@/stores/xapi-collection.store";
+import { defineStore } from "pinia";
+import { computed } from "vue";
+
+export const useAlarmStore = defineStore("alarm", () => {
+  const messageCollection = useXapiCollectionStore().get("message");
+
+  function subscribe() {
+    const subscription = messageCollection.subscribe({ immediate: false });
```

Comment:
Why is it necessary to defer the subscription?

Reply:
I don't remember if you were there when we talked about the alarms.

They are quite heavy to load, so we decided to load them only when the user requests it.

With this option, we can subscribe and get access to the subscription computed's and ref's but load the data later.